Assets/Mirror/Runtime/NetworkReader.cs

```diff
@@ -53,17 +53,17 @@ public byte[] ReadBytesAndSize()
 
         // http://sqlite.org/src4/doc/trunk/www/varint.wiki
         // NOTE: big endian.
-        public UInt32 ReadPackedUInt32()
+        public uint ReadPackedUInt32()
         {
-            UInt64 value = ReadPackedUInt64();
-            if (value > UInt32.MaxValue)
+            ulong value = ReadPackedUInt64();
```

Comment:
is this really better? there is some value in knowing that it's a 32 bit int, especially since it's networking

Reply:
I can see it both ways.   

On one hand it is not obvious that UInt64 and ulong are aliases,  Most people just don't know.  so ulong make sense because it is what everyone uses.

On the other hand,  the byte size really matters when it comes to the serializer,  and it is not obvious that ulong is 64 bits unless you know your C#.  

So which one is more readable?  I don't know,  honestly.